NewHorizons/External/Modules/PropModule.cs

```diff
@@ -210,6 +210,11 @@ public class DetailInfo
             /// Should this detail stay loaded even if you're outside the sector (good for very large props)
             /// </summary>
             public bool keepLoaded;
+
+            /// <summary>
+            /// Should this object dynamically move around
+            /// </summary>
+            public bool hasPhysics;
```

Comment:
The name hasPhysics sounds a little weird to me. What about just “physics”?

Reply:
hasPhysics sounds better to me